pkg/client-sdk/store/sql/migration/20250624211039_init.up.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE IF NOT EXISTS vtxo (
 	vout INTEGER NOT NULL,
 	script TEXT NOT NULL,
 	amount INTEGER NOT NULL,
-	commitment_txid TEXT NOT NULL,
+	commitment_txids TEXT NOT NULL,
```

Comment:
postgres has an `ARRAY` type, maybe worth to use it ?

Reply:
https://sqlite.org/vtab.html#table_valued_functions

Reply:
Not sure we need to do query tho. If we always return all the list all the time (no granular filtering) and never update that field often, may be ok cc/ @Kukks